datafusion/functions/benches/ltrim.rs

```diff
@@ -141,8 +141,8 @@ fn run_with_string_type<M: Measurement>(
         ),
         |b| {
             b.iter(|| {
-                #[allow(deprecated)] // TODO use invoke_batch
-                black_box(ltrim.invoke(&args))
+                #[allow(deprecated)] // TODO use invoke_with_args
```

Comment:
likewise here I think we can remove the #allow

Reply:
Comment below